apis/Google.Cloud.DevTools.Common/docs/history.md

```diff
@@ -0,0 +1,10 @@
+# Version history
+
+# Version 1.1.0, released 2019-12-09
+
+- [Commit 518728a](https://github.com/googleapis/google-cloud-dotnet/commit/518728a): A few minor tidy-ups of comments
```

Comment:
Is this something that we'd want in version history? I would say no.

Reply:
I'd normally agree, but it shows that at least *something* has happened other than the dependency versions being updated :)